src/command_line_parser.h

```diff
@@ -144,8 +144,6 @@ struct PerfAnalyzerParameters {
   size_t warmup_request_count{0};
 
   size_t session_concurrency{0};
-  SessionConcurrencyMode session_concurrency_mode{
-      SessionConcurrencyMode::Disabled};
```

Comment:
Why is this removed?

Reply:
There was a merge conflict but accepting the change still contained this part, so I had to remove it manually to match with the main. There's no more SessionConcurrencyMode in main branch.